src/assets/locales/en/translation.json

```diff
@@ -37,7 +37,14 @@
     },
     "entityModal": {
       "details": "{{title}} Details",
-      "of": "of"
+      "contract_details": "Contract Details",
```

Comment:
@developer0623, is there a better way to do this?

Reply:
@anonymoussprocket if we make the several entities' views, it is good way.

Reply:
I meant passing in "Contract" instead somehow.